src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -279,7 +291,11 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
 
             if (result.getParseErrorMessage() == null) {
-                processTree(result.getTree());
+                final DetailNode javadocTree = result.getTree();
+                if (doesCheckAcceptJavadocWithUnclosedHtmlTag()
+                        || !JavadocUtils.hasUnclosedTag(javadocTree)) {
```

Comment:
@ps-sp , @Vladlis , @rnveach 

I forgot that javadoc support is still hold in each Javadoc Check with sharing ASTs between Checks by static variable - hack.
we need to keep this flag in that static collection of ASTs for javadoc. So Check before execution/traverse could make a decision.
So each javadoc Check should have property - `violateExecutionOnNonTightHtml`. Default value is FALSE, as I think most Checks will not care about this.

Property `skipNonTightHtml` will be unclear to user, as skipped execution is always mean 2 states: bug or by configuration. It is uncertain.